### PR TITLE
chore: bump production dependencies

### PR DESCRIPTION
**Pull request by erisu**
### Motivation and Context

Update the third-party production npm dependencies to ensure that our package remains vulnerability free that can come from third-party dependencies.

Enables us to use new features, leverage speed improvements, and resolve issues that were caused by third-party dependency bugs.

### Description

* bplist-parser@^0.2.0
* elementtree@^0.1.7
* endent@^1.3.0
* fs-extra@^8.1.0
* glob@^7.1.6
* minimatch@^3.0.4
* underscore@^1.9.1

### Testing

- `npm t`

### Checklist

- [x] I've run the tests to see all new and existing tests pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,18 +24,18 @@
...
     "cross-spawn": "^6.0.5",
-    "elementtree": "0.1.7",
-    "endent": "^1.1.1",
-    "fs-extra": "^8.0.0",
-    "glob": "^7.1.2",
-    "minimatch": "^3.0.0",
+    "elementtree": "^0.1.7",
+    "endent": "^1.3.0",
+    "fs-extra": "^8.1.0",
+    "glob": "^7.1.4",
```
Comment (raphinesse) [inline package.json]:
```suggestion
    "glob": "^7.1.6",
```

File: package.json [hunk 2]
```diff
@@ -24,18 +24,18 @@
...
+    "minimatch": "^3.0.4",
     "plist": "^3.0.1",
     "q": "^1.4.1",
     "strip-bom": "^4.0.0",
-    "underscore": "^1.8.3",
-    "which": "^1.3.0"
+    "underscore": "^1.9.1",
+    "which": "^1.3.1"
```
Comment (raphinesse) [inline package.json]: There already is `which@2`, but if we remove it in #102 this will not matter anymore

File: package.json [hunk 3]
```diff
@@ -24,18 +24,18 @@
   },
   "dependencies": {
     "ansi": "^0.3.1",
-    "bplist-parser": "^0.1.0",
+    "bplist-parser": "^0.2.0",
     "cross-spawn": "^6.0.5",
```
Comment (raphinesse) [inline package.json]: There already is `cross-spawn@7`, but if we remove it in #98 this will not matter anymore

Comment (erisu) [reply package.json]: 👍

File: package.json (hunk 2 above)
Comment (erisu) [reply package.json]: 👍

**Pull request merged**